docs/02-guides/server-side-events/create-an-integration.mdx

```diff
@@ -7,7 +7,135 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+This guide will help you to create an integration that will dispatch events to external services.
```

Comment:
**issue:** duplicate from the Front-matter description

![2024-02-08_16-01](https://github.com/front-commerce/developers.front-commerce.com/assets/75968/8e164990-ecb9-4554-b569-a07dc244b6ca)

**suggestion:** remove